src/Camelotia.Presentation.Wpf/Views/FolderView.xaml

```diff
@@ -0,0 +1,40 @@
+﻿<UserControl 
+    x:Class="Camelotia.Presentation.Wpf.Views.FolderView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+    xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
+    xmlns:designTime="clr-namespace:Camelotia.Presentation.DesignTime;assembly=Camelotia.Presentation"
+    d:DataContext="{d:DesignInstance Type=designTime:DesignTimeFolderViewModel, IsDesignTimeCreatable=True}"
+    d:Background="#FF9FA8DA"
+    d:Height="40"
+    mc:Ignorable="d" >
+    <Grid>
+        <Button BorderThickness="0" Background="Transparent" Command="{Binding Provider.SetPath}" CommandParameter="{Binding FullPath}" HorizontalContentAlignment="Stretch" VerticalContentAlignment="Stretch" Padding="0">
+            <DockPanel Background="Transparent" Margin="0">
+                <Menu DockPanel.Dock="Right" Background="Transparent">
+                    <MenuItem x:Name="TopLevelMenu" ItemsSource="{Binding Children}" Height="{Binding ActualHeight, RelativeSource={RelativeSource AncestorType={x:Type Menu}}}" Padding="0">
```

Comment:
Probably worth removing the background at all? So it looks like this:

<img src="https://user-images.githubusercontent.com/6759207/92997162-d97c2200-f519-11ea-86bf-d06aca8bcdfc.png" width="450" />

```suggestion
                    <MenuItem x:Name="TopLevelMenu" ItemsSource="{Binding Children}" Height="{Binding ActualHeight, RelativeSource={RelativeSource AncestorType={x:Type Menu}}}" Padding="0" Background="Transparent">
```

Reply:
I was unsure wether to make it transparent or have it stand out a bit but I think the way you suggested it looks better :+1: